packages/react/src/patterns/sub-patterns/ContentItem/README.md

```diff
@@ -50,8 +55,16 @@ ReactDOM.render(<App />, document.querySelector('#app'));
 | --------- | -------- | --------- | ------------- | -------------------------------- |
 | `heading` | NO       | String    | null          | Heading text                     |
 | `copy`    | NO       | String    | null          | Copy text                        |
+| `cta`     | NO       | object    | null          | CTA object. See `cta` below.     |
 | `image`   | NO       | object    | null          | Image object. See `image` below. |
 
+### cta (optional)
+
+| Name   | Data Type | Description                            |
+| ------ | --------- | -------------------------------------- |
+| `type` | String    | Type of CTA (jump, local, or external) |
+| `copy` | String    | Text of the CTA component              |
```

Comment:
I think th emore important thing we outline in the documentation is the `style` being used, and possibly `types` allowed and then we can link to the  `CTA` component for documentation on how to use thosee styles and types.